src/string.cpp

```diff
@@ -494,12 +494,13 @@ bool strtolower(std::string &str, std::string::size_type offs)
 bool IsValidChar(WChar key, CharSetFilter afilter)
 {
 	switch (afilter) {
-		case CS_ALPHANUMERAL:  return IsPrintable(key);
-		case CS_NUMERAL:       return (key >= '0' && key <= '9');
-		case CS_NUMERAL_SPACE: return (key >= '0' && key <= '9') || key == ' ';
-		case CS_ALPHA:         return IsPrintable(key) && !(key >= '0' && key <= '9');
-		case CS_HEXADECIMAL:   return (key >= '0' && key <= '9') || (key >= 'a' && key <= 'f') || (key >= 'A' && key <= 'F');
-		default: NOT_REACHED();
+	case CS_ALPHANUMERAL:   return IsPrintable(key);
+	case CS_NUMERAL:        return (key >= '0' && key <= '9');
+	case CS_NUMERAL_SPACE:  return (key >= '0' && key <= '9') || key == ' ';
+	case CS_NUMERAL_SIGNED: return (key >= '0' && key <= '9') || key == '-';
+	case CS_ALPHA:          return IsPrintable(key) && !(key >= '0' && key <= '9');
+	case CS_HEXADECIMAL:    return (key >= '0' && key <= '9') || (key >= 'a' && key <= 'f') || (key >= 'A' && key <= 'F');
+	default: NOT_REACHED();
```

Comment:
Indentation seems to have gotten wrong here. (Maybe check your editor settings.)

Reply:
Ok thanks. Yes, vscode wanted to format the code in a completely different way so I had to turn off auto format here.